filter_plugins/list-dict-diff.py

```diff
@@ -8,9 +8,62 @@ def get_remaining_items(list_a, list_b, key_a, key_b):
 
     return list_a
 
+# For manage-workflow-templates role
+# Takes a workflow defined in a dictionary and processes it into a list
+def flatten_workflow_nodes(workflows):
+    flattened_workflow, flattened_start_tree = [], []
+    current_node_index = 1
+
+    # Processes each node beginning at start and all their success/failure branches
+    for workflow in workflows:
```

Comment:
Just a bit of a personal preference - try to stay away from variables named identical with just a `s` difference. In this case, I'd just name the list something like `workflow_list`. In that way, it's easier to differentiate from the `workflow` variable.